tests/run-macros/tasty-overload-secondargs/Test_2.scala

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Nitpick: Could you remove the extra whitespaces on the top and bottom of this test?

Reply:
dome